joblib/testing.py

```diff
@@ -51,6 +68,24 @@ def assert_raises_regex(expected_exception, expected_regexp,
                                      expected_exception(expected_regexp))
 
 
+try:
```

Comment:
just had a look at scikit-learn (who did the same recently) and there's nothing of this kind, just
```
from nose.tools import assert_raises
```